### DISCUSSION TITLE
Hologram doesn't display

### DISCUSSION THREAD

**Discussion by Smidgemork** [maruohon/litematica#584] [Q&A]
Hello!

I was wondering if someone could help me out here.  When I create schematics in my world in the latest version and paste my schematics in they show as solid blocks which makes it hard to see what I am building (it also doesn't show where I have placed wrong blocks, etc.).

In previous versions the blue shaded hologram was set automatically so I could see what needed to be placed or what is placed wrong.  I've inserted a screenshot below with the original build on the left and the schematic on the right.  Is there a feature I may have accidentally enabled?

Thanks

![image](https://user-images.githubusercontent.com/113837739/190901806-e6035314-d841-46f6-9ba8-5682e70591d1.png)

**Comment by maruohon** [comment]
Look through your `Visuals` config category for configs you may have changed. There are options to disable the entire colored schematic overlay, or the individual types/colors. Also if you use shaders, depending on the shader pack it may break some or all rendering in the mod.
